app/views/posts/show.html.erb

```diff
@@ -1 +1,18 @@
-<h1> This is a post</h1>
\ No newline at end of file
+<div main-container>
+<div class="post-text"> 
+<div class="user-info">
+<h1>Post n°<%= @post.id %> by <%=@user.name %></h1>
+<p>Comments: <%= @post.comments_counter %></p>
+<p>Likes: <%= @post.likes_counter %></p>
+</div>
+<p><%= @post.text %></p>
+</div>
+<div class="comment-section">
+<% if @post.comments.any? %>
+      <%= render 'shared/comments', post: @post %>
+    <% else %>
+      <h3>No hay comentarios</h3>
+    <% end %>
+        
+</div>
+</div>
```

Comment:
- [ ] [OPTIONAL] It will be nice to properly indent these lines of code in order to have a better and clean code base.
you should have :
``` ruby
<div main-container>
  <div class="post-text">
    <div class="user-info">
      <h1>Post n°<%= @post.id %> by <%= @user.name %></h1>
      <p>Comments: <%= @post.comments_counter %></p>
      <p>Likes: <%= @post.likes_counter %></p>
    </div>
    <p><%= @post.text %></p>
  </div>
  <div class="comment-section">
    <% if @post.comments.any? %>
      <%= render 'shared/comments', post: @post %>
    <% else %>
      <h3>No hay comentarios</h3>
    <% end %>
  </div>
</div>
```